### PR TITLE
feature/make-more-cross-platformy: replacing *nix specifics

### PR DESCRIPTION
**Pull request by f4bio**
### package.json

- replace *nix's `rm` with node-native alternative `trash`
- removing inline variable (`NODE_ENV`)

builds without any problems on:
```

                                ..,   f4bio@d3sktop
                    ....,,:;+ccllll   -------------
      ...,,+:;  cllllllllllllllllll   OS: Windows 10 Pro x86_64
,cclllllllllll  lllllllllllllllllll   Host: ASUS All Series
llllllllllllll  lllllllllllllllllll   Kernel: 3.0.7-338.x86_64
llllllllllllll  lllllllllllllllllll   Uptime: 45 mins
llllllllllllll  lllllllllllllllllll   Packages: 35 (scoop)
llllllllllllll  lllllllllllllllllll   Shell: bash 4.4.23
llllllllllllll  lllllllllllllllllll   Resolution: 3440x1440
                                      DE: Aero
llllllllllllll  lllllllllllllllllll   WM: Explorer
llllllllllllll  lllllllllllllllllll   WM Theme: Custom
llllllllllllll  lllllllllllllllllll   CPU: Intel i7-4790K (8) @ 3.991GHz
llllllllllllll  lllllllllllllllllll   GPU: NVIDIA GeForce GTX 1060 6GB
llllllllllllll  lllllllllllllllllll   Memory: 13048MiB / 24507MiB
`'ccllllllllll  lllllllllllllllllll
       `' \*::  :ccllllllllllllllll
                       ````''*::cll
                                 ``

```

### CODE REVIEW

**Comment by Dreamacro** [comment]
Usually, we use `rimraf` instead of `trash-cli` 🤔

**Comment by f4bio** [comment]
> Usually, we use `rimraf` instead of `trash-cli` 🤔

Oh... was probably on autopilot when making those changes (personally I prefer using `del` and `trash`. But, you're right of course, `rimraf` is way more commonly used)
Sorry about that, I will updated this PR asap!

File: package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     },
     "dependencies": {
         "axios": "^0.19.0",
+        "build": "^0.1.4",
```
Comment (Dreamacro) [inline package.json]: Is it a mistake? 🤔

File: package.json [hunk 2]
```diff
@@ -17,8 +17,8 @@
     },
     "homepage": "https://github.com/Dreamacro/clash-dashboard",
     "scripts": {
-        "build": "npm run clean-dist && NODE_ENV=production webpack --config=configs/webpack/prod.js",
-        "clean-dist": "rm -rf dist",
+        "build": "npm run clean-dist && webpack --config=configs/webpack/prod.js",
+        "clean-dist": "npx rimraf ./dist",
```
Comment (Dreamacro) [inline package.json]: if `rimraf` in devDependencies, we don't need `npx` 🤓

**Pull request closed without merging**